doc/manual/constructors.rst

```diff
@@ -586,7 +586,7 @@ to another, you should probably define a ``convert`` method instead.
 On the other hand, if your constructor does not represent a lossless
 conversion, or doesn't represent "conversion" at all, it is better
 to leave it as a constructor rather than a ``convert`` method.  For
-example, the ``Array(Int)`` constructor creates a zero-dimensional
+example, the ``Array{Int}`` constructor creates a zero-dimensional
```

Comment:
Thanks for pointing it out. I will go through everything and look for these mistakes. I'm wondering, though. Should it be `Array(Int)` or `Array{Int}()` ? I've changed it to the latter in the actual code.